sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeUriBuilder.cs

```diff
@@ -310,33 +359,6 @@ private RequestUriBuilder BuildUri()
             };
         }
 
-        /// <summary>
-        /// Get the account name from the domain portion of a Uri.
-        /// </summary>
-        /// <param name="uri">The Uri.</param>
-        /// <param name="serviceSubDomain">The service subdomain used to validate that the
-        /// domain is in the expected format. This should be "blob" for blobs, "file" for files,
-        /// "queue" for queues, "blob" and "dfs" for datalake.</param>
-        /// <returns>Account name or null if not able to be parsed</returns>
-        internal static string GetAccountNameFromDomain(Uri uri, string serviceSubDomain)
-        {
-            var accountEndIndex = uri.Host.IndexOf(".", StringComparison.InvariantCulture);
-            if (accountEndIndex >= 0)
-            {
-                var serviceStartIndex = accountEndIndex + 1;
-                var serviceEndIndex = uri.Host.IndexOf(".", serviceStartIndex, StringComparison.InvariantCulture);
-                if (serviceEndIndex > serviceStartIndex)
-                {
-                    var service = uri.Host.Substring(serviceStartIndex, serviceEndIndex - serviceStartIndex);
-                    if (service == serviceSubDomain)
-                    {
-                        return uri.Host.Substring(0, accountEndIndex);
-                    }
-                }
-            }
-            return null;
-        }
-
         /// <summary>
         /// If path starts with a slash, remove it
         /// </summary>
```

Comment:
We can also remove the following GetPath and IsHostIPEndPointStyle methods because there are extensions available.  I added temporary copies here to work around visibility issues with that last quick DataLake preview.

Reply:
removed.